### PR TITLE
Add rustfmt gitignore

### PR DESCRIPTION
**Pull request by waynenilsen**
**Reasons for making this change:**

I keep modifying my gitignore to include this entry

**Links to documentation supporting these rule changes:** 

This seems straightforward to me, not sure it is written out anywhere.

### CODE REVIEW

File: Rust.gitignore [hunk 1]
```diff
@@ -5,3 +5,6 @@
 # Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
 # More information here http://doc.crates.io/guide.html#cargotoml-vs-cargolock
 Cargo.lock
+
+# These are backup files generated by rustfmt
+*.rs.bk
```
Comment (shiftkey) [inline Rust.gitignore]: The [official gitignore](https://github.com/rust-lang-nursery/rustfmt/commit/5ac67adb409f374aa37a40860649f8c770ecdee8) for rustfmt has a slightly different rule: `**/*.rs.bk` - want to mimic that here?

**Comment by waynenilsen** [comment]
My git-fu is not exactly as good as it could be not sure how to "squash" these commits.. or is it "rebase"? That is if you want me to.

**Comment by hojatdelfan** [comment]
thankshttp://www.goftare.com/

**Comment by hojatdelfan** [comment]
[thanks!](http://www.goftare.com/)

**Comment by shiftkey** [comment]
@waynenilsen it's fine, I can do that from here. Thanks!

**Pull request merged**